docs/how-to/push-update-on-mainnet.md

```diff
@@ -0,0 +1,120 @@
+Overview
+========
+
+Software update is a protocol mechanism that allows nodes to agree on software changes without altering protocol constants and update to this software version.
+1. Software update is proposed. Proposal is a datatype that gets into
+   the blockchain. It contains information about version changes and
+   hashes of update files.
+2. Update files are uploaded to the S3 bucket.
+3. Software update is confirmed by voting from majority of nodes.
+4. Nodes that see an update try to download and apply it
+   automatically.
+
+Now, again, step-by-step.
+
+Prerequisites
+=============
+1. Cluster nodes' keys. They are crucial to have since cluster nodes
+   are the only nodes that have stake in the system so they are the
+   only nodes that can propose an update and vote for it.
+2. Access to the S3 bucket
+3. Installers corresponding to software version we're pushing to cluster
+   - Those are two executables (exe for win, pkg for mac) which are to be provided by QA (after QA procedures passed)
```

Comment:
Are these executables the same sort our CI systems are creating? I imagine updates should all be signed using the same keys we currently use for both the standard Daedalus installers.

Reply:
> Are these executables the same sort our CI systems are creating?

AFAIK yes.

Reply:
@fersel should know.

Reply:
@volhovM here we need a clarification that those two executables are installers provided by our CI.